BlueTracker.SDK.Performance/Core/ApiWrapper.cs

```diff
@@ -19,21 +19,27 @@ public abstract class ApiWrapper
     {
         private readonly string _serverAddress;
         private readonly string _authorization;
+        private static HttpClient _httpClient;
+
+        /// <summary>
+        /// Singleton instance for every ApiClient to prevent depletion of connection pool
+        /// </summary>
+        private static HttpClient HttpClient => _httpClient ?? (_httpClient = new HttpClient());
```

Comment:
The static client can be constructed in a single line.
`private static HttpClient _httpClient = new HttpClient();`